2023/articles/2023-12-23.pod

```diff
@@ -0,0 +1,295 @@
+Author: olaf@wundersolutions.com
+Title: Elves Versus Unused Imports
+Topic: App::perlimports
+
+=encoding utf8
+
+Santa's elves are interested in code quality. This is something we touched on
+about 9 years ago in L<How Santa's Elves Keep their Workshop
+Tidy|https://perladvent.org/2014/2014-12-16.html> and, more recently, in
+L<Elves Versus Typos|https://perladvent.org/2023/2023-12-21.html>. Since the
+elves like keeping up with the times, they are currently experimenting with
+another, newer Perl code quality tool. It's finally time to take a close look
+at
+L<perlimports|https://metacpan.org/dist/App-perlimports/view/script/perlimports>.
+L<perlimports|https://metacpan.org/dist/App-perlimports/view/script/perlimports>
+is a fixer -- it can rewrite your code for you. It's also a linter, so you can
+use it to report on problems without fixing them. With this power comes great
+responsibility and C<perlimports> tries to be responsible. Inspired by
+L<goimports|https://pkg.go.dev/golang.org/x/tools/cmd/goimports>,
+C<perlimports> is an opinionated tool which tries to force its opinions on your
+code. What you get in return is a tool which takes much of the burden of
+managing imports out of your hands.
+
+If you're interested in an in-depth discussion of how C<perlimports> and Perl's
+C<use> and C<require> work, check out L<perlimports or "Where did that symbol come
+from?"|https://www.youtube.com/watch?v=fKqxdTbGxYY>.
```

Comment:
It's very much a personal preference but I was reading the rendered POD and followed this link to test it for 404 etc. and was surprised it was a youtube video instead of an article/paper/blog post. It might be kind to those on low bandwidth or metered connections (or grumpy folks like me) just to indicate it is an auto-playing video here. eg. "check out the [link] video"?